### PR TITLE
Task/APPS-2591 — update CRC module for care integration

### PR DESCRIPTION
**Pull request by giuliacampana**
Minor updates that make CRC usable within CARE app.

### CODE REVIEW

File: src/containers/restitutionreferral/RestitutionReferral.js [hunk 1]
```diff
@@ -222,14 +225,16 @@ const RestitutionReferral = ({ personId } :Props) => {
...
-              aria-label="Success Button"
-              color="success"
-              onClick={goToProfile}
-              variant="outlined">
-            Back to Profile
-          </Button>
+          <SubmissionFieldsGrid>
+            <Typography gutterBottom>Submitted!</Typography>
+            <Button
+                aria-label="Success Button"
```
Comment (solomontang) [inline src/containers/restitutionreferral/RestitutionReferral.js:231]: According to MDN
> The `aria-label` attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use `aria-labelledby` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

File: src/containers/repairharm/RepairHarmAgreement.js [hunk 2]
```diff
@@ -197,14 +200,16 @@ const RepairHarmAgreement = ({ personId } :Props) => {
...
-              aria-label="Success Button"
-              color="success"
-              onClick={goToProfile}
-              variant="outlined">
-            Back to Profile
-          </Button>
+          <SubmissionFieldsGrid>
+            <Typography gutterBottom>Submitted!</Typography>
+            <Button
+                aria-label="Success Button"
```
Comment (solomontang) [inline src/containers/repairharm/RepairHarmAgreement.js:206]: According to MDN
> The `aria-label` attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use `aria-labelledby` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

File: src/containers/peacemaker/PeacemakerInformationForm.js [hunk 3]
```diff
@@ -140,6 +147,20 @@ const PeacemakerInformationForm = ({ personId } :Props) => {
           onSubmit={onSubmit}
           schema={schema}
           uiSchema={uiSchema} />
+      {isSuccess(submitRequestState) && (
+        <CardSegment>
+          <SubmissionFieldsGrid>
+            <Typography gutterBottom>Submitted!</Typography>
+            <Button
+                aria-label="Success Button"
```
Comment (solomontang) [inline src/containers/peacemaker/PeacemakerInformationForm.js:155]: According to MDN
> The `aria-label` attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use `aria-labelledby` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

File: src/containers/intake/IntakeForm.js [hunk 4]
```diff
@@ -225,14 +229,16 @@ const IntakeForm = ({ personId } :Props) => {
...
-              aria-label="Success Button"
-              color="success"
-              onClick={goToProfile}
-              variant="outlined">
-            Back to Profile
-          </Button>
+          <SubmissionFieldsGrid>
+            <Typography gutterBottom>Submitted!</Typography>
+            <Button
+                aria-label="Success Button"
```
Comment (solomontang) [inline src/containers/intake/IntakeForm.js:235]: According to MDN
> The `aria-label` attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use `aria-labelledby` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

File: src/containers/restitutionreferral/RestitutionReferral.js (hunk 1 above)
Comment (giuliacampana) [reply src/containers/restitutionreferral/RestitutionReferral.js:231]: oh whoops, ya i guess the `aria-labelledby` is what i should be using in these cases.

**Review by solomontang — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**